jquery.highlight.js

```diff
@@ -115,16 +207,16 @@
     } else {
       if (settings.svgPathStyle === 'circle') {
         pathFunc = svgCirclePath;
+      } else if (settings.svgPathStyle === 'radius') {
+        pathFunc = generateSvgRadiusPath(settings.radius);
       }
     }
 
     els.each(function() {
-      offset = $(this).offset();
-      top = offset.top;
-      left = offset.left;
-      bottom = top + $(this).outerHeight();
-      right = left + $(this).outerWidth();
-      path += pathFunc(top, left, bottom, right);
+      if (!isElementVisible($(this))) { return; }
```

Comment:
no need to wrap `this` as a jQuery element. `isElementVisible` can handle non-jQuery els.